src/coreclr/src/vm/ceeload.cpp

```diff
@@ -11312,6 +11312,21 @@ LPCWSTR GetManagedCommandLine()
     return s_pCommandLine;
 }
 
+LPCWSTR GetCommandLineForDiagnostics()
+{
+    // Get the managed command line.
+    LPCWSTR pCmdLine = GetManagedCommandLine();
+
+    // Checkout https://github.com/dotnet/coreclr/pull/24433 for more information about this fall back.
+    if (pCmdLine == nullptr)
+    {
+        // Use the result from GetCommandLineW() instead
+        pCmdLine = GetCommandLineW();
```

Comment:
I see this is just a refactoring to omit duplicated code, which is great. Curious question. Is it handled somewhere else the case where `GetCommandLineW()` is also a `nullptr`? Or does this case never happen at all?

Reply:
As far as I understand, `GetCommandLineW()` should *never* return NULL.